11/problem11.rb

```diff
@@ -0,0 +1,4 @@
+puts "Please Enter your number:"
+num=Integer(gets.chomp!)
+
+puts num>1 && num<10 ? "True" : "False"
```

Comment:
`number.between?(min, max)` can also be used to tell if a number is in range of min and max.

Reply:
`(0..10).include?(num)`